### DISCUSSION TITLE
4mmc + Amphetamines (incl MDMA)

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#84] [Drug Interactions]
So far it looks like 4mmc in combination with other amphetamines (including MDMA) can increase neurotoxicity, specifically on dopamine nerve endings (not serotonin!).

**note that these are animal (mice) studies**

Angoa-Pérez, M., Kane, M. J., Briggs, D. I., Francescutti, D. M., Sykes, C. E., Shah, M. M., Thomas, D. M., & Kuhn, D. M. (2013). Mephedrone does not damage dopamine nerve endings of the striatum, but enhances the neurotoxicity of methamphetamine, amphetamine, and MDMA. Journal of Neurochemistry, 125(1), 102–110. https://doi.org/10.1111/jnc.12114

Angoa-Pérez, M., Kane, M. J., Herrera-Mundo, N., Francescutti, D. M., & Kuhn, D. M. (2014). Effects of combined treatment with mephedrone and methamphetamine or 3,4-methylenedioxymethamphetamine on serotonin nerve endings of the hippocampus. Life Sciences, 97(1), 31–36. https://doi.org/10.1016/j.lfs.2013.07.015

**Comment by utaninja** [comment]
MDMA + 4mmc - https://github.com/TripSit/drugs/issues/52
Caffeine + 4mmc - https://github.com/TripSit/drugs/issues/57#ref-pullrequest-2128716685

These are very interesting studies you've shared.
Would both go for caffeine and mdma then? Since both affect dopamine? I also believe studies for both of these interactions listed above would go for 4mmc + Amphetamine aswell.

**Comment by utaninja** [comment]
All have been pushed - https://github.com/TripSit/drugs/issues/309
